src/02-room-details.js

```diff
@@ -49,8 +84,29 @@ function getRoomByDinosaurName(dinosaurs, rooms, dinosaurName) {}
       "Kit Hopkins Education Wing"
     ]
  */
-function getConnectedRoomNamesById(rooms, id) {}
+//return a string arr with name of room that's connected to a given room
+//use rooms.connectsTo
+// return error message 
+// helper function const room = getRoomById(rooms, ‘sldkfwei’)
+// gets room name by ID
+
+function getConnectedRoomNamesById(rooms, id) {
+  let destination = []; //array that's being returned
 
+  for(let room of rooms){ // loops through room array
+    for (const connect of room.connectsTo){ //loops through connectsTo array
+      if(connect === id && connect !== 'incorrect-id'){ //checks given id against inputted id
```

Comment:
This is hardcoded. If I pass the id 'uhfe7199879' as an argument your function should then return `Room with ID of 'uhfe7199879' could not be found.`; Knowing this how would you update your code?